portal/src/pages/my/privacy.vue

```diff
@@ -146,19 +150,54 @@ export default {
     },
     submitButtonLabel() {
       if (
+        !this.$store.getters.hasGivenCommunityPermission &&
+        this.hasInitialCommunityPermission
+      ) {
+        return this.$t('Delete-account-and-login')
+      } else if (
         this.$store.getters.hasGivenCommunityPermission !==
         this.hasInitialCommunityPermission
       ) {
         return this.$t('save-privacy-settings-and-logout')
       }
       return this.$t('save-privacy-settings')
+    },
+    submitButtonStyle() {
```

Comment:
Kan je niet beter die `warning` class conditioneel maken in het template stuk?

Reply:
ok